### PR TITLE
ENH remove unused zip keys

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@isuruf this should fix things!

**Comment by isuruf** [comment]
Can you remove lines 9-10?

**Comment by beckermr** [comment]
done!

**Comment by beckermr** [comment]
any more changes needed @conda-forge/core?

**Comment by beckermr** [comment]
thanks @isuruf !

File: recipe/migrations/pypy.yaml [hunk 1]
```diff
@@ -31,13 +29,3 @@ python_impl:
   - cpython    # [not (osx and arm64)]
   - cpython
   - pypy       # [not (win64 or (osx and arm64))]
-
-
-zip_keys:
-  -
```
Comment (mariusvniekerk) [inline recipe/migrations/pypy.yaml:37]: This one matters for the py39 migration.....

Comment (mariusvniekerk) [reply recipe/migrations/pypy.yaml:37]: ah right this is from pypy

**Pull request merged**